Pocket/pocket-ios
# Unified Home

## Summary
* This PR replaces the legacy Home recommendations with Unified Home. The new Api is used to fetch slates and recommendations. The UI structure remains the same.

## References 
* See ticket number in the branch name

## Implementation Details
* PocketGraph: introduce new `Corpus` entities
* Update `SlateService`, replace existing lineup query with `homeSlateLineup`
* Core Data: map `SlateLineup`, `Slate` and `Recommendation` to corresponding `Corpus` entities

> **Note**
Analytics updates and Core Data entity renaming will happen in a separate PR


## Test Steps
* Build/run this branch
* Ho to Home and make sure you see the unified home slates (Today's Pocket Hits, For you, etc)
* Make sure recommendations are readable and actionable as they should be (e.g. save)
* Make sure Home works as expected from a UI standpoint

## PR Checklist:

## Screenshots

<p align=center>

<img height=500 src=https://github.com/Pocket/pocket-ios/assets/34376330/5d7f9f82-c400-4062-8ae4-4b968ab66f6f>
</p>
